src/zenml/orchestrators/local/local_orchestrator.py

```diff
@@ -75,6 +76,11 @@ def run_pipeline(
         if runtime_configuration is None:
             runtime_configuration = RuntimeConfiguration()
 
+        if runtime_configuration.schedule:
+            logger.warning("Local Orchestrator currently does not support the"
```

Comment:
This same warning should also happen for the `AirflowOrchestrator` right?

Reply:
Implemented true scheduling for airflow now - non-schduled airflow runs will run once immediately